### ISSUE TITLE
Telemetry: Noob Singleplayer Replays

### ISSUE THREAD

**Issue by CrazyEddieTK** [ZeroK-RTS/Chobby#320]
If a player has less than X games played, Chobby (or something else?) should upload their singleplayer replays so that developers can review their first X games. This will help with:

- Calibrating AI difficulties and behavior
- Finding problems with tutorials and missions
- Better understanding how new players react to and learn Zero-K

The uploads should include both completed and exited games. They should include tutorials, missions, and skirmishes.

The noob uploads should be stored with a flag that hides them from the standard replay search function and only allows developers and admins to search for them, see them, or download them.

**Comment by gajop** [comment]
I totally agree with the desire here, although it might be non-trivial to implement.
Users should be able to play offline, and I'm not sure if it should be uploaded without consent. "Developers spying on noobs" can be an undesired backlash.

**Comment by sprunk** [comment]
Popup: "hey, we'd like to collect newbie feedback, upload replay? Y/N"
